release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1230,6 +1230,13 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 
 [id="ocp-4-12-bug-fixes"]
 == Bug fixes
+
+[discrete]
+[id="ocp-4-12-api-auth-bug-fixes"]
+==== API Server and Authentication
+
+* Previously, the Cluster Authentication Operator state was set to `progressing = false` after receiving a `workloadIsBeingUpdatedTooLong` error. At the same time, `degraded = false` was kept for the time of the `inertia` defined. Consequently, the shortened amount of progressing and increased time of degradedation would create a situation where `progressing = false` and `degraded = false` were set prematurely. This caused inconsistent OpenShift CI tests because a healthy state was assumed, which was incorrect. This issue has been fixed by removing the `progressing = false` setting after the `workloadIsBeingUpdatedTooLong` error is returned. Now, because there is no `progressing = false` state, OpenShift CI tests are more consistent. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2111842#h11[*BZ2111842*])
```

Comment:
```suggestion
* Previously, the Cluster Authentication Operator state was set to `progressing = false` after receiving a `workloadIsBeingUpdatedTooLong` error. At the same time, `degraded = false` was kept for the time of the `inertia` defined. Consequently, the shortened amount of progressing and increased time of degradedation would create a situation where `progressing = false` and `degraded = false` were set prematurely. This caused inconsistent OpenShift CI tests because a healthy state was assumed, which was incorrect. This issue has been fixed by removing the `progressing = false` setting after the `workloadIsBeingUpdatedTooLong` error is returned. Now, because there is no `progressing = false` state, OpenShift CI tests are more consistent. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2111842[*BZ2111842*])
```

Reply:
I think the anchor in this link needs to be removed bc it takes you to the comment area of the bugzilla.